### PR TITLE
console script for running pyhdx

### PR DESCRIPTION
**Pull request by sajetan**
this is for https://github.com/Jhsmit/PyHDX/issues/165

### CODE REVIEW

File: pyhdx/panel/configurations.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import os
+import configparser
+
+class ConfigurationSettings(object):
+    def load_config(self):
+        self.config = configparser.ConfigParser()
+        self.config.read((os.path.join(os.path.dirname(__file__), '', self.config_file)))
```
Comment (Jhsmit) [inline pyhdx/panel/configurations.py:10]: Please use `Path` from `pathlib` for path/file operations, such as in for example `test_fileIO.py` 
(eg `directory = Path(__file__).parent` )

File: pyhdx/panel/configurations.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import os
+import configparser
+
+class ConfigurationSettings(object):
```
Comment (Jhsmit) [inline pyhdx/panel/configurations.py:7]: Add a numpy-style class-level docstring describing `__init__` args and their types

File: pyhdx/panel/configurations.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+        self.config.read((os.path.join(os.path.dirname(__file__), '', self.config_file)))
+
+    def __init__(self, config_file='config.ini'):
+        self.config_file = config_file
+        self.load_config()
+
+    def load_cluster(self):
+        return str(self.config.get('cluster','ip'))+":"+str(self.config.get('cluster','port'))
+
+    def update_cluster(self, ip, port):
```
Comment (Jhsmit) [inline pyhdx/panel/configurations.py:21]: Add docstrings to each method

**Comment by Jhsmit** [comment]
I've created a clean py38 environment and checkout out this branch, installed dependencies and installed pyhdx with 
`$ python setup.py install`

I get this error, which i think might be because `cli.py` is in the wrong directory?

  File "C:\Users\jhsmi\Miniconda3\envs\py38_pydhx_pr_191\Scripts\pyhdx-script.py", line 33, in <module>
    sys.exit(load_entry_point('PyHDX==0.3.1.dev171', 'console_scripts', 'pyhdx')())
  File "C:\Users\jhsmi\Miniconda3\envs\py38_pydhx_pr_191\Scripts\pyhdx-script.py", line 25, in importlib_load_entry_point
    return next(matches).load()
  File "C:\Users\jhsmi\Miniconda3\envs\py38_pydhx_pr_191\lib\site-packages\importlib_metadata\__init__.py", line 166, in load
    module = import_module(match.group('module'))
  File "C:\Users\jhsmi\Miniconda3\envs\py38_pydhx_pr_191\lib\importlib\__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 965, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'cli'

**Comment by sajetan** [comment]
> I get this error, which i think might be because cli.py is in the wrong directory?
works fine in my setup, could be due to your PYTHONPATH?

File: requirements.txt [hunk 4]
```diff
@@ -1,3 +1,4 @@
+pbr==5.5.1
```
Comment (Jhsmit) [inline requirements.txt:1]: Why is this exact requirement needed? 
Is it exactly 5.5.1 or is >=5.5.1 also OK?

**Review by Jhsmit — COMMENTED** [review]
AUTHORS and ChangeLog are AFAIK autogenerated by pbr and should not be added to git

**Comment by Jhsmit** [comment]
I get this error when I try to run PyHDX with `$ pyhdx serve`:

This is on windows in a conda env with python 3.8 and pbr 5.5.1
Running `$ python serve.py` does work

```
Traceback (most recent call last):
  File "C:\Users\jhsmi\Miniconda3\envs\py38_pydhx_pr_191\lib\site-packages\pkg_resources\__init__.py", line 582, in _build_master
    ws.require(__requires__)
  File "C:\Users\jhsmi\Miniconda3\envs\py38_pydhx_pr_191\lib\site-packages\pkg_resources\__init__.py", line 899, in require
    needed = self.resolve(parse_requirements(requirements))
  File "C:\Users\jhsmi\Miniconda3\envs\py38_pydhx_pr_191\lib\site-packages\pkg_resources\__init__.py", line 790, in resolve
    raise VersionConflict(dist, req).with_context(dependent_req)
pkg_resources.ContextualVersionConflict: (cloudpickle 1.4.1 (c:\users\jhsmi\appdata\roaming\python\python37\site-packages), Requirement.parse('cloudpickle>=1.5.0'), {'distributed'})

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\jhsmi\Miniconda3\envs\py38_pydhx_pr_191\Scripts\pyhdx-script.py", line 33, in <module>
    sys.exit(load_entry_point('PyHDX==0.3.1.dev177', 'console_scripts', 'pyhdx')())
  File "C:\Users\jhsmi\Miniconda3\envs\py38_pydhx_pr_191\Scripts\pyhdx-script.py", line 25, in importlib_load_entry_point
    return next(matches).load()
  File "C:\Users\jhsmi\Miniconda3\envs\py38_pydhx_pr_191\lib\site-packages\importlib_metadata\__init__.py", line 166, in load
    module = import_module(match.group('module'))
  File "C:\Users\jhsmi\Miniconda3\envs\py38_pydhx_pr_191\lib\importlib\__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 953, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "C:\Users\jhsmi\Miniconda3\envs\py38_pydhx_pr_191\lib\site-packages\pyhdx\__init__.py", line 2, in <module>
    import setuptools   # Import prevents warning
  File "C:\Users\jhsmi\Miniconda3\envs\py38_pydhx_pr_191\lib\site-packages\setuptools\__init__.py", line 19, in <module>
    import setuptools.version
  File "C:\Users\jhsmi\Miniconda3\envs\py38_pydhx_pr_191\lib\site-packages\setuptools\version.py", line 1, in <module>
    import pkg_resources
  File "C:\Users\jhsmi\Miniconda3\envs\py38_pydhx_pr_191\lib\site-packages\pkg_resources\__init__.py", line 3267, in <module>
    @_call_aside
  File "C:\Users\jhsmi\Miniconda3\envs\py38_pydhx_pr_191\lib\site-packages\pkg_resources\__init__.py", line 3251, in _call_aside
    f(*args, **kwargs)
  File "C:\Users\jhsmi\Miniconda3\envs\py38_pydhx_pr_191\lib\site-packages\pkg_resources\__init__.py", line 3280, in _initialize_master_working_set
    working_set = WorkingSet._build_master()
  File "C:\Users\jhsmi\Miniconda3\envs\py38_pydhx_pr_191\lib\site-packages\pkg_resources\__init__.py", line 584, in _build_master
    return cls._build_from_requirements(__requires__)
  File "C:\Users\jhsmi\Miniconda3\envs\py38_pydhx_pr_191\lib\site-packages\pkg_resources\__init__.py", line 597, in _build_from_requirements
    dists = ws.resolve(reqs, Environment())
  File "C:\Users\jhsmi\Miniconda3\envs\py38_pydhx_pr_191\lib\site-packages\pkg_resources\__init__.py", line 785, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'pyct>=0.4.4' distribution was not found and is required by panel

```

File: requirements.txt (hunk 4 above)
Comment (sajetan) [reply requirements.txt:1]: Yes I think it should be okay, I was just testing and left it like that

**Comment by sajetan** [comment]
>pkg_resources.DistributionNotFound: The 'pyct>=0.4.4' distribution was not found and is required by panel

I can add pyct in the requirements if that helps!

**Comment by Jhsmit** [comment]
There is something else going on, my console entry script isnt installing correctly, some kind of interaction with pkg_resources
I do have the dependency because `$python serve.py` runs without issues.

**Pull request merged**